tests/unit/db/feed_tests.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# Copyright (c) 2016 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+feed module - Unit tests for Feed class
+"""
+
+import unittest
+from requests import Session
+import json
+import os
+
+from cloudant.feed import Feed
+from cloudant.document import Document
+from cloudant.design_document import DesignDocument
+from cloudant.error import CloudantArgumentError, CloudantException
+from cloudant._2to3 import unicode_
+
+from .unit_t_db_base import UnitTestDbBase
+from ... import BYTETYPE
+
+class FeedTests(UnitTestDbBase):
+    """
+    Feed unit tests
+    """
+
+    def setUp(self):
+        """
+        Set up test attributes
+        """
+        super(FeedTests, self).setUp()
+        self.db_set_up()
+        self.db_updates_url = '/'.join([self.client.cloudant_url, '_db_updates'])
+        self.changes_url = '/'.join([self.db.database_url, '_changes'])
+        self.session = self.client.r_session
+        self.cloudant_test = os.environ.get('RUN_CLOUDANT_TESTS') is not None
+
+    def tearDown(self):
+        """
+        Reset test attributes
+        """
+        self.db_tear_down()
+        super(FeedTests, self).tearDown()
+
+    def test_constructor_changes(self):
+        """
+        Test constructing a _changes feed
+        """
+        feed = Feed(self.session, self.changes_url, raw_data=True,
+            chunk_size=1, feed='continuous')
+        self.assertEqual(feed._url, self.changes_url)
+        self.assertIsInstance(feed._r_session, Session)
+        self.assertTrue(feed._raw_data)
+        self.assertDictEqual(feed._options, {'feed': 'continuous'})
+        self.assertEqual(feed._chunk_size, 1)
+
+    def test_constructor_db_updates(self):
+        """
+        Test constructing a _db_updates feed
+        """
+        feed = Feed(self.session, self.db_updates_url, feed='continuous',
```

Comment:
My plan with `_db_updates` tests was to add them in a follow up PR since this one is already quite bloated.  We won't make a release without the tests being in place.  I am tracking them with #147.